app/request_list.tsx

```diff
@@ -85,11 +94,23 @@ const RequestList = (props: RequestObj) => {
       showPagination={true}
       showPaginationTop={false}
       showPaginationBottom={true}
-      pageSize={10}
-      freezeWhenExpanded
+      defaultPageSize={10}
+      page={props.PageDetails[props.tabId] ? props.PageDetails[props.tabId].currentPageNumber : 0}
```

Comment:
should the default pageNumber 0 or 1?

Reply:
The page numbers start with `0`. It uses `index`. However, the pageNumber is displayed as `1` on view.